src/thing/remove.ts

```diff
@@ -169,6 +171,37 @@ export const removeDatetime: RemoveOfType<Date> = (thing, property, value) => {
   );
 };
 
+/**
+ * Create a new Thing with the given datetime removed for the given Property.
+ *
+ * The original `thing` is not modified; this function returns a cloned Thing with updated values.
+ *
+ * @param thing Thing to remove a datetime value from.
+ * @param property Property for which to remove the given datetime value.
+ * @param value Time to remove from `thing` for the given `property`.
+ * @returns A new Thing equal to the input Thing with the given value removed for the given Property.
+ */
+export const removeTime: RemoveOfType<Time> = (thing, property, value) => {
+  internal_throwIfNotThing(thing);
+  return removeLiteralMatching(
+    thing,
+    property,
+    xmlSchemaTypes.time,
+    function (foundTime) {
+      const deserializedTime = deserializeTime(foundTime);
+      if (deserializedTime) {
+        return (
+          deserializedTime.hour === value.hour &&
+          deserializedTime.minute === value.minute &&
+          deserializedTime.second === value.second
```

Comment:
Shouldn't we also check the milliseconds and time zone offsets?

Reply:
I omitted them because they're not always guaranteed, may make sense to have the check for those which do tho

Reply:
> Overall looks good, but I think it's important to preserve the timezone data, if given.
> 
> Btw, you checked the "The changelog has been updated" checkbox, but this feature isn't listed there yet - same goes for the `Date` support. Could you add that too?

Where can I find the changelog??

Reply:
In the root of the repository there is a file `CHANGELOG.md`. You can look at the existing entries for the format, it's roughly in line with https://keepachangelog.com/.